augur/ancestral.py

```diff
@@ -399,7 +399,7 @@ def run(args):
         aln = args.alignment
         ref = None
         if args.root_sequence:
-            for fmt in ['fasta', 'genbank']:
+            for fmt in [BIOPYTHON_FASTA_FORMAT, 'genbank']:
```

Comment:
We could abstract away this change here by doing the version switch inside of `read_sequence(s)` helpers instead.

Reply:
Done:

https://github.com/nextstrain/augur/blob/5c5325605f575ed86fd36211758573991450d986/augur/io/sequences.py#L20-L22